tools/run_backups

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -x
+cd /home/backups || exit 1
+echo $(date +%T) : Starting backups of jenkins config.xml files ...
+ssh jenkins@ci.adoptium.net 'ls -1 /home/jenkins/.jenkins/plugins/*.?pi | while read FILENAME; do basename $FILENAME ; unzip -p $FILENAME META-INF/MANIFEST.MF | sort -d | grep "Short-Name\|Long-Name\|Plugin-Version" ; echo --- ; done > /home/jenkins/.jenkins/pluginVersions.txt'
+#ssh jenkins@ci.adoptopenjdk.net 'tar czf - $(find . -name config.xml )' > jenkins/jenkins.$(date +%Y%m%d).tar.gz
```

Comment:
commented out code?

Reply:
That was due to a very recent change where I wanted to keep the older version, but happy to remove it.